ietf-scitt-charter.md

```diff
@@ -4,6 +4,8 @@ Over the years, rapid technological advancements have motivated organizations to
 While these improvements help organizations increase efficiency and swiftly bring innovations to market, the rapid increase in scale, size, and complexity of supply chains has led to more frequent and sophisticated supply chain attacks.
 The traditional methods of safeguarding supply chains (e.g., pre- and post-audit methodologies) are no longer adequate.
 
+Scitt forms a building block that will allow implemeters to compose complex supply chain integrity systems. For example, public computer interface system could report its software composition and that can be compared against known software compositions for such a device, recorded in a public ledger, thus providing an individual using the system with confidence that it will behave as expected, when expected, and nothing/nowhen else.
+
 Problem Statement
 =================
 It is challenging to manage the compliance of products across end-to-end, global supply chains. 
```

Comment:
Please update the problem statement to match the BOF Problem statement

Reply:
Will do in another PR

Reply:
Minor suggestion, please consider replacing "and nothing/nowhen else." WITH 
"consistently, without deviation."

Reply:
Saving 1st comment in issue & adding 2nd comment via commit.